src/sentry/grouping/strategies/newstyle.py

```diff
@@ -52,8 +58,11 @@
     "colno",
 ]
 
+# TODO(markus)
+StacktraceEncoderReturnValue = Any
```

Comment:
We will type this whenever we get around to adding types to similarity-v2 (grouping-based similarity) though it's unclear if we may want to remove that feature entirely.